src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -73,6 +73,8 @@ internal sealed partial class BoundLambda : IBoundLambdaOrFunction
 
         internal InferredLambdaReturnType InferredReturnType { get; }
 
+        internal bool InAnonymousFunctionConversion { get; private set; }
```

Comment:
Not sure if I should instead use `BoundNode._attributes` for this flag? But that would require this property to live in `BoundNode` as field `_attributes` is private.